internal/info/info_test.go

```diff
@@ -293,68 +302,292 @@ func TestDetail_Get(t *testing.T) {
+							URL:      "https://github.com/golang/go/blob/" + runtime.Version() + "/src/info.go#L100",
+							FuncName: "github.com/vdaas/vald/internal/info.TestDetail_Get",
+							File:     runtime.GOROOT() + "/src/info.go",
+							Line:     100,
+						},
+					},
+					PrepOnce: func() sync.Once {
+						o := sync.Once{}
+						o.Do(func() {})
+						return o
+					}(),
+				},
+			},
+		},
+		{
+			name: "get return detail object with go mod path",
+			beforeFunc: func() {
+				i := 0
+				rtCaller = func(skip int) (pc uintptr, file string, line int, ok bool) {
+					if i == 0 {
+						i++
+						return uintptr(0), "/tmp/go/pkg/mod/github.com/vdaas/vald/internal/info.go", 100, true
+					}
+					return uintptr(1), "info.go", 100, false
+				}
+				rtFuncForPC = func(uintptr) *runtime.Func {
+					return runtime.FuncForPC(reflect.ValueOf(TestDetail_Get).Pointer())
+				}
+			},
+			afterFunc: func() {
+				rtCaller = runtime.Caller
+				rtFuncForPC = runtime.FuncForPC
+			},
+			want: want{
+				want: Detail{
+					GitCommit: "master",
+					GoVersion: runtime.Version(),
+					GoOS:      runtime.GOOS,
+					GoArch:    runtime.GOARCH,
+					StackTrace: []StackTrace{
+						StackTrace{
+							URL:      "https://github.com/vdaas/vald/internal/info.go#L100",
+							FuncName: "github.com/vdaas/vald/internal/info.TestDetail_Get",
+							File:     "/tmp/go/pkg/mod/github.com/vdaas/vald/internal/info.go",
+							Line:     100,
+						},
+					},
+					PrepOnce: func() sync.Once {
+						o := sync.Once{}
+						o.Do(func() {})
+						return o
+					}(),
+				},
+			},
+		},
+		{
+			name: "get return detail object with go mod path with version",
+			beforeFunc: func() {
+				i := 0
+				rtCaller = func(skip int) (pc uintptr, file string, line int, ok bool) {
+					if i == 0 {
+						i++
+						return uintptr(0), "/tmp/go/pkg/mod/github.com/vdaas@v0.0.0-20171023180738-a3a6125de932/vald/internal/info.go", 100, true
+					}
+					return uintptr(1), "info.go", 100, false
+				}
+				rtFuncForPC = func(uintptr) *runtime.Func {
+					return runtime.FuncForPC(reflect.ValueOf(TestDetail_Get).Pointer())
+				}
+			},
+			afterFunc: func() {
+				rtCaller = runtime.Caller
+				rtFuncForPC = runtime.FuncForPC
+			},
+			want: want{
+				want: Detail{
+					GitCommit: "master",
+					GoVersion: runtime.Version(),
+					GoOS:      runtime.GOOS,
+					GoArch:    runtime.GOARCH,
+					StackTrace: []StackTrace{
+						StackTrace{
+							URL:      "https://github.com/vdaas/blob/v0.0.0-20171023180738-a3a6125de932/vald/internal/info.go#L100",
+							FuncName: "github.com/vdaas/vald/internal/info.TestDetail_Get",
+							File:     "/tmp/go/pkg/mod/github.com/vdaas@v0.0.0-20171023180738-a3a6125de932/vald/internal/info.go",
+							Line:     100,
+						},
+					},
+					PrepOnce: func() sync.Once {
+						o := sync.Once{}
+						o.Do(func() {})
+						return o
+					}(),
+				},
+			},
+		},
+		{
+			name: "get return detail object with go mod path with 2 version",
```

Comment:
```suggestion
			name: "return detail object when go mod path with 2 version is set",
```

Reply:
where can I confirm the 2 version set?

Reply:
sorry it is not 2 version, it is called `pseudo-version`. 
ref: https://blog.golang.org/publishing-go-modules

```suggestion
			name: "return detail object when go mod path contains pseudo version",
```